src/mscorlib/src/System/Runtime/Intrinsics/X86/Avx.PlatformNotSupported.cs

```diff
@@ -197,36 +197,36 @@ public static class Avx
         /// <summary>
         /// __int8 _mm256_extract_epi8 (__m256i a, const int index)
         /// </summary>
-        public static sbyte ExtractSByte<T>(Vector256<T> value, byte index) where T : struct { throw new PlatformNotSupportedException(); }
+        public static sbyte Extract(Vector256<sbyte> value, byte index) { throw new PlatformNotSupportedException(); }
```

Comment:
@jkotas, you requested many similar functions be made generic in the ARM proposal. Should the same be done here and elsewhere in the API surface?

Reply:
AFAIR The idea was to use generic for functions with parameter types comprising majority or all supported types. In hte case of `Extract` functions there is only limited number of overloads.

Reply:
Right. Here is a link to the discussion for reference: https://github.com/dotnet/corefx/issues/26179#issuecomment-355615654

Reply:
`Extract` is one that operates on 8/10 of the supported `numeric` types, I could considered that a majority.

In either case, it would be good to get a final statement from @jkotas/@CarolEidt so we don't need yet another pr